main.js

```diff
@@ -78,20 +82,20 @@ function increaseGuessCounter() {
 
 //Check Guess checks the challengers guesses and iterates through the array. Then the
 function checkGuess(challengerGuesses) {
-  for(var i=0; i<challengerGuesses.length; i++){
+  for (var i = 0; i < challengerGuesses.length; i++) {
     var challengerValue = parseInt(challengerGuesses[i].value);
-    if(challengerValue<currentGame.currentCorrectNumber){
+    if (challengerValue < currentGame.currentCorrectNumber) {
       guessHelpText[i].innerHTML = "that's too low";
     } else if (challengerValue > currentGame.currentCorrectNumber) {
       guessHelpText[i].innerHTML = "that's too high";
-    } else {
+    } else if(challengerValue == currentGame.currentCorrectNumber){
+      currentGame.hasBeenWon = true;
       guessHelpText[i].innerHTML = "BOOM!";
-      if (i==0) {
-         currentGame.winner=challenger1NameValue.value;
-      } else {
-        currentGame.winner=challenger2NameValue.value;
-      };
-      gameWon();
+      if (i == 0) {
+        currentGame.winner = challenger1NameValue.value;
+      } else if(i==1){
```

Comment:
I'm glad you added this specific else if statement, could save us some headache down the road.